include/motor-control/core/motor_messages.hpp

```diff
@@ -34,6 +37,7 @@ struct Ack {
     uint8_t seq_id;
     uint32_t current_position;
     AckMessageId ack_id;
+    bool lim_sw_triggered;
```

Comment:
The ack message gets put into a `MoveCompleted` command which is found in `move_status_reporter_task.hpp`